Vision/PID/diagramskill/debug.py

```diff
@@ -0,0 +1,25 @@
+from PIL import Image
+from io import BytesIO
+import cv2 
+from azure.storage.blob import BlobServiceClient
+import numpy as np
+
+def write_with_bounding(img,boxes):
+    color = (200,0,0)
+    for b in boxes:  
+        cv2.rectangle(img, (b.topleft_x, b.topleft_y), (b.bottomright_x,b.bottomright_y), color, 3)
+
+    #if circles is not None:

```

Comment:
Why is this commented out?

Reply:
This is only for testing and is not needed by customers. I just left it in if someone wants to modify the code and needed the scaffolding